sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 3.4.0-beta.1 (Unreleased)
+- Fix bug that user-name-attribute cannot be configured.(#20209)
```

Comment:
add link to this issue?

Reply:
Also, should we add this under a `bug fixes` section?

Reply:
@saragluna , updated.